src/theme/theme.ts

```diff
@@ -0,0 +1,17 @@
+import { breakpoints } from './breakpoints'
+import { mediaQueries } from './mediaQueries'
+import { colors } from './colors'
+
+const theme = {
+  colors,
+  fonts: {
+    title: 'Poppins, sans-serif',
+    body: 'Roboto, sans-serif',
+  },
+  space: [0, 5, 10, 15, 20],
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 20](https://app.codacy.com/app/PrideInLondon/pride-london-web/pullRequest?prid=4694377)